akka-remote/src/main/resources/reference.conf

```diff
@@ -1162,6 +1163,15 @@ akka {
           # situations where things locate each other by hostname, in scenarios where host names are dynamic
           # and not known up front it can make sense to have this disabled.
           hostname-verification = off
+
+          # How long will the SSLContext instance will be cached for. Defaults to 10 days.
```

Comment:
No need to mention the default here, since the real value is just a few lines down.